lib/app/resources/resourcefiles_test.go

```diff
@@ -341,25 +334,7 @@ spec:
       volumes:
         - name: data
           hostPath:
-            path: /data
----
-apiVersion: batch/v2alpha1
```

Comment:
Should the CronJob be removed or just changed to a later API version?

Reply:
Duh - you're right, reverting.